config.yaml

```diff
@@ -20,26 +20,46 @@ options:
     type: string
     default: ""
     description: >-
-      The private docker registry configured as dockerhub mirror to be used by the runners. If set 
-      a message will be displayed prior to job execution on self-hosted runner informing users to 
-      use the provided registry.
+      The URL to the private docker registry configured as the dockerhub mirror to be used by the 
+      runners. If set a message will be displayed prior to job execution on self-hosted runner 
+      informing users to use the provided registry. You must use https:// as the protocol.
   experimental-openstack-clouds-yaml:
     type: string
     default: ""
     description: >-
       (Experimental, under development) The openstack clouds.yaml
       (https://docs.openstack.org/python-openstackclient/pike/configuration/index.html#clouds-yaml).
       Setting the openstack-clouds-yaml would enable spawning runners on OpenStack, instead of LXD
-      vms. This feature is not yet ready and is under development.
+      vms. Providing a valid clouds.yaml will enable OpenStack integration. This feature is not yet
+      ready and is under development.
+  experimental-openstack-flavor:
+    type: string
+    default: ""
+    description: |
+      (Experimental, under development) The openstack flavor to use to spawn virtual machines.
+      This feature is not yet ready and is under development.
+  experimental-openstack-network:
+    type: string
+    default: ""
+    description: |
+      (Experimental, under development) The openstack network for virtual machines. This feature
+      is not yet ready and is under development.
+  experiential-openstack-image-build-unit:
+    type: int
+    default: -1
+    description: >-
+      (Experimental, tmp workaround, will be removed in the future) The unit that performs the
+      OpenStack image build. All other units will have the image build disabled. Cannot be changed
+      after deployment. Will only take effect with OpenStack integration enabled.
```

Comment:
Should these still be experimental?

Reply:
except for the build unit which should get removed soon

Reply:
Updated the configs.